src/build/prepareElectronApp.ts

```diff
@@ -17,58 +17,75 @@ function pickElectronAppArgs(options: AppOptions): any {
   return {
     accessibilityPrompt: options.nativefier.accessibilityPrompt,
     alwaysOnTop: options.nativefier.alwaysOnTop,
+    appBundleId: options.packager.appBundleId,
+    appCategoryType: options.packager.appCategoryType,
     appCopyright: options.packager.appCopyright,
     appVersion: options.packager.appVersion,
+    arch: options.packager.arch,
+    asar: options.packager.asar,
     backgroundColor: options.nativefier.backgroundColor,
     basicAuthPassword: options.nativefier.basicAuthPassword,
     basicAuthUsername: options.nativefier.basicAuthUsername,
+    blockExternalUrls: options.nativefier.blockExternalUrls,
     bounce: options.nativefier.bounce,
     browserwindowOptions: options.nativefier.browserwindowOptions,
+    buildDate: new Date().getTime(),
     buildVersion: options.packager.buildVersion,
     clearCache: options.nativefier.clearCache,
     counter: options.nativefier.counter,
     crashReporter: options.nativefier.crashReporter,
     darwinDarkModeSupport: options.packager.darwinDarkModeSupport,
+    derefSymlinks: options.packager.derefSymlinks,
     disableContextMenu: options.nativefier.disableContextMenu,
     disableDevTools: options.nativefier.disableDevTools,
     disableGpu: options.nativefier.disableGpu,
+    disableOldBuildWarning: options.nativefier.disableOldBuildWarning,
     diskCacheSize: options.nativefier.diskCacheSize,
+    download: options.packager.download,
+    electronVersionUsed: options.packager.electronVersion,
     enableEs3Apis: options.nativefier.enableEs3Apis,
+    executableName: options.packager.executableName,
     fastQuit: options.nativefier.fastQuit,
     fileDownloadOptions: options.nativefier.fileDownloadOptions,
     flashPluginDir: options.nativefier.flashPluginDir,
     fullScreen: options.nativefier.fullScreen,
     globalShortcuts: options.nativefier.globalShortcuts,
     height: options.nativefier.height,
+    helperBundleId: options.packager.helperBundleId,
     hideWindowFrame: options.nativefier.hideWindowFrame,
     ignoreCertificate: options.nativefier.ignoreCertificate,
     ignoreGpuBlacklist: options.nativefier.ignoreGpuBlacklist,
     insecure: options.nativefier.insecure,
     internalUrls: options.nativefier.internalUrls,
-    blockExternalUrls: options.nativefier.blockExternalUrls,
-    maxHeight: options.nativefier.maxHeight,
+    isUpgrade: options.packager.upgrade,
+    junk: options.packager.junk,
```

Comment:
Is this a leftover of a dev test?

Reply:
No it's a built-in electronPackagaer option. I just wanted to add it here in case it was important for the rebuild:

```
/**
     * Ignores [system junk files](https://github.com/sindresorhus/junk) when copying the Electron app,
     * regardless of the [[ignore]] option.
     *
     * **Note:** `junk` will have no effect if the [[prebuiltAsar]] option is set.
     */
    junk?: boolean;
```